### ISSUE TITLE
x/mobile/cmd/gomobile: fork/exec arm-linxu-androideabi-nm: no such file or directory

### ISSUE THREAD

**Issue by EtienneBruines** [golang/go#16268]
# TL;DR
`gomobile build` results in:

>
gomobile: /home/travis/gopath/pkg/gomobile/android-ndk-r12/arm/bin/arm-linux-androideabi-nm /tmp/gomobile-work-903719122/lib/armeabi-v7a/libadventure.so: fork/exec /home/travis/gopath/pkg/gomobile/android-ndk-r12/arm/bin/arm-linux-androideabi-nm: no such file or directory

# What version of Go are you using (`go version`)?
`go version go1.6.2 linux/amd64`

#  What operating system and processor architecture are you using (`go env`)?
```
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/travis/gopath"
GORACE=""
GOROOT="/home/travis/.gimme/versions/go1.6.2.linux.amd64"
GOTOOLDIR="/home/travis/.gimme/versions/go1.6.2.linux.amd64/pkg/tool/linux_amd64"
GO15VENDOREXPERIMENT="1"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0"
CXX="g++"
CGO_ENABLED="1"
```

# What did you do?

We're running tests on travis, including testing if `gomobile build` succeeds. This is the error that occurs:

```
gomobile: /home/travis/gopath/pkg/gomobile/android-ndk-r12/arm/bin/arm-linux-androideabi-nm /tmp/gomobile-work-903719122/lib/armeabi-v7a/libadventure.so: fork/exec /home/travis/gopath/pkg/gomobile/android-ndk-r12/arm/bin/arm-linux-androideabi-nm: no such file or directory
```

Related travis build log: https://travis-ci.org/EngoEngine/engo/builds/142017612#L373

# What did you expect to see?
Expected build to pass. 

# What did you see instead?
Build failed.

**Comment by axengine** [comment]
我下载了android-ndk-r12b-linux-x86_64.zip,将对应的文件拷贝到pkg/gomobile/android-ndk-r12对应目录下，可以正常编译。

**Comment by Hovelander** [comment]
谢谢！That led me to the fix that appears to be in progress:

https://go-review.googlesource.com/#/c/24724/

**Comment by tnzk** [comment]
@axengine Hi, can you share what you exactly did for a workaround? I have tried downloading NDK r12b and put it in `$GOPATH/pkg/gomobile` as `android-ndk-r12` but did not work.

**Comment by Hovelander** [comment]
Still getting "/pkg/gomobile/android-ndk-r12/arm/bin/arm-linux-androideabi-nm: no such file or directory" even after applying Elias' latest patches @ https://go-review.googlesource.com/#/c/24724/ locally.  No "b" joy to be had.

Tried mucking about with the suggestions from @axengine & @hyangah, but no joy from my efforts there either (although take those efforts with a large, ham fisted flavor of newbie salt).

Guess I'll just turn my attention to the working iOS example until this is sorted and wait patiently for Elias to figure out the issue.  (Good luck and thanks Elias & others!)

**Comment by hyangah** [comment]
Sorry for the delay. 
gomobile NDK release is blocked in some internal process.
The easiest workaround meanwhile is to roll back to d5b8fb1623bebcf0646aba78cfd8208159e1cc45 (https://go.googlesource.com/mobile/+/34d472ecb61e8fb5396dfa0b2bc3048e780f76d7)

I will update this bug once the NDK release is approved.

**Comment by hyangah** [comment]
they are uploaded. 
@eliasnaur you can submit the cl.

**Comment by Hovelander** [comment]
Works as intended, thanks!

**Comment by telecoda** [comment]
Great work 👍
